### DISCUSSION TITLE
Multiple lists in a single post

### DISCUSSION THREAD

**Discussion by neurosnap** [neurosnap/lists.sh#15] [Ideas]
One thing I've seen come up a few times is that people would like to be able to create multiple lists within a single post.  One way we could accomplish this is by:
- Blank lines close the previous list and start a new one
- Headers close the previous list and start a new one

I'm curious what others thoughts are on this idea?

Implementation: https://github.com/neurosnap/lists.sh/pull/16

**Comment by neurosnap** [comment]
cc @bentsai -- curious of your thoughts

**Reply by bentsai** [reply]
I think a new list when you see a header makes sense since semantically a header implies hierarchy, and a higher one at that.

This also removes the ability to put headers as list items, which seems reasonable for the same reason of hierarchy.

**Reply by bentsai** [reply]
This also gives you one kind of nested lists.
